x-pack/plugins/fleet/server/routes/package_policy/index.ts

```diff
@@ -112,11 +217,32 @@ export const registerRoutes = (router: FleetAuthzRouter) => {
   router.versioned
     .post({
       path: PACKAGE_POLICY_API_ROUTES.CREATE_PATTERN,
+      // TODO missing fleetAuthz?
```

Comment:
thanks for adding this comment, we definitely have to add it!

Reply:
actually found that this was intentional, the authz check moved to the service here: https://github.com/elastic/kibana/pull/140458